tools/docker/Dockerfile.jinja

```diff
@@ -104,6 +108,77 @@ RUN rm -rf sysbench-1.0.20 \
+    && unzip sqlite-amalgamation-3460100.zip \
+    && cd sqlite-amalgamation-3460100 \
+    && gcc -g -shared -fPIC -c sqlite3.c \
+    && gcc -g -shared -fPIC -o libsqlite3.so sqlite3.o \
+    && mv ./libsqlite3.so /lib/x86_64-linux-gnu/ \
+    && mv ./sqlite3.h /usr/include/x86_64-linux-gnu/ \
+    && mv ./sqlite3ext.h /usr/include/x86_64-linux-gnu/
+
+# Instal LevelDB 1.23
+WORKDIR /root
+RUN mkdir -p /usr/local/leveldb/benchmark/
+RUN git clone -b 1.23 --recurse-submodules https://github.com/google/leveldb.git \
+    && cd leveldb \
+    && mkdir -p build \
+    && cd build \
+    && cmake -DCMAKE_BUILD_TYPE=Release .. \
+    && cmake --build . \
+    && make install \
+    && mv ./db_bench /usr/local/leveldb/benchmark/ \
+    && mv ./db_bench_sqlite3 /usr/local/leveldb/benchmark/ 
+
+# Install Redis-7.0.15
+WORKDIR /root
+RUN wget https://download.redis.io/releases/redis-7.0.15.tar.gz \
+    && tar -xzvf redis-7.0.15.tar.gz \
+    && cd redis-7.0.15 \
+    && make -j \
+    && make PREFIX=/usr/local/redis install 
+
+# Install Nginx only with http enabled
+WORKDIR /root
+RUN wget https://nginx.org/download/nginx-1.26.2.tar.gz \
+    && tar -xzvf nginx-1.26.2.tar.gz \
+    && cd nginx-1.26.2 \
+    && ./configure --with-cc-opt="-O2" --with-ld-opt="-static" --without-select_module --without-poll_module \
+    --without-quic_bpf_module --without-http_charset_module --without-http_gzip_module --without-http_ssi_module \
+    --without-http_userid_module --without-http_access_module --without-http_auth_basic_module --without-http_mirror_module \
+    --without-http_geo_module --without-http_map_module --without-http_split_clients_module --without-http_referer_module \
+    --without-http_rewrite_module --without-http_proxy_module --without-http_fastcgi_module --without-http_uwsgi_module \
+    --without-http_scgi_module --without-http_grpc_module --without-http_memcached_module --without-http_limit_conn_module \
+    --without-http_limit_req_module --without-http_empty_gif_module --without-http_browser_module --without-http_upstream_hash_module \
+    --without-http_upstream_ip_hash_module --without-http_upstream_least_conn_module --without-http_upstream_random_module \
+    --without-http_upstream_keepalive_module --without-http_upstream_zone_module --without-http-cache --without-mail_pop3_module \
+    --without-mail_imap_module --without-mail_smtp_module --without-stream_limit_conn_module --without-stream_access_module \
+    --without-stream_geo_module --without-stream_map_module --without-stream_split_clients_module --without-stream_return_module \
+    --without-stream_pass_module --without-stream_set_module --without-stream_upstream_hash_module --without-stream_upstream_least_conn_module \
+    --without-stream_upstream_random_module --without-stream_upstream_zone_module --without-pcre --without-pcre2
+
+WORKDIR /root/nginx-1.26.2
+RUN make -j \
+    && make install
+
+# Clear cached files
+WORKDIR /root
```

Comment:
Move the remove logic close to the code that adds or generates the temp files. This way, when we review or remove, let's say, ngnix, from the Dockerfile, we can simply examine or remove the whole code block regarding ngnix (instead of two code blocks).